source/adapters/cuda/kernel.cpp

```diff
@@ -377,3 +377,8 @@ urKernelSetArgSampler(ur_kernel_handle_t hKernel, uint32_t argIndex,
   }
   return Result;
 }
+
+UR_APIEXPORT ur_result_t UR_APICALL urKernelSetSpecializationConstants(
```

Comment:
Would it be worth it to do the same for HIP?

Reply:
HIP already does this.

Reply:
Thank you